TD_Antiking.txt

```diff
@@ -0,0 +1,41 @@
+﻿
```

Comment:
Try to use English language only, especially for comments. The reasons are the following:
1. you never know who will be your teammates in future and English is the only international language;
2. your English will be never good enough (in comparison to native speakers) so you'll always need some practice;
3. writing your thoughts in English makes them better formulated;
4. some environments and coding tools don't support cyrillic symbols and your language will look like something so much weird, that even computers do not know what it is.